tower-request-modifier/src/lib.rs

```diff
@@ -10,13 +10,20 @@ use std::sync::Arc;
 use tower_service::Service;
 
 /// Wraps an HTTP service, injecting authority and scheme on every request.
-#[derive(Clone)]
-pub struct RequestModifier<T, B> {
+pub struct RequestModifier<T, B>
+where
+    T: Clone,
```

Comment:
You should not need `where` bounds here. The `Clone` bound can be isolated to the `Clone` impl